apps/settings/js/panels/root/panel.js

```diff
@@ -158,3 +133,28 @@ define(function(require) {
     });
   };
 });
+
+/**
+ * Update the sim and NFC related items based on mozMobileConnections and Device.
+ */
+
+var updateSimItemsAndNfc = function rph_refrehsSimItems() {
```

Comment:
I'm not fully understand the reason, why this function was initially added to the 'root' panel (instead of some panel related to connectivity settings), but the current change is obviously wrong. If you need some functionality not related to the 'Panel' interface please consider creation of new module as described in "settings/README.md"